src/middlewared/middlewared/plugins/gluster_linux/peer.py

```diff
@@ -125,4 +157,60 @@ def pool(self, job):
         including localhost.
         """
 
-        return self.__peer_wrapper(peer.pool)
+        # get the local viewpoint of the remote peers in the TSP
+        local_view = self.__peer_wrapper(peer.status)
+        if not local_view:
+            return local_view
```

Comment:
Good use of assignemnt expression

if not local_view := self.__peer_wrapper(peer.status):
   return

:P